SpatialGDK/Documentation/content/get-started/gdk-and-starter-project.md

```diff
@@ -0,0 +1,70 @@
+# Get started: 3 - Set up the Unreal GDK module and Starter Project
+
+Follow the steps below to:
+
+* Clone the Starter Project repository.
+* Clone the Unreal GDK into the `Plugins` folder of your game.
+* Build the Unreal GDK module dependencies which the Starter Project needs so it can work with the GDK.
+* Run the Starter Project on SpatialOS
+
+### Cloning
+
```

Comment:
Add an explanation that you are cloning the GDK repository into the already-cloned Starter Project repository coz that's a bit of an odd one, IMHO.
So something like:
"In this step, you are going to clone the Starter Project repository and then clone the GDK repository into your copy of the Starter Project. Note that the GDK needs to go into the Starter Project's `Plugins` directory."

Reply:
It just rams it home .

Reply:
agreed